src/main/resources/org/jenkinsci/plugins/github_branch_source/GitHubSCMSource/config-detail.jelly

```diff
@@ -2,45 +2,40 @@
 <j:jelly xmlns:j="jelly:core" xmlns:c="/lib/credentials"
          xmlns:f="/lib/form"
          xmlns:scm="/jenkins/scm/api/form">
+  <j:choose>
+    <j:when test="${descriptor.apiUriSelectable}">
+    <f:entry title="${%API endpoint}" field="apiUri">
+      <f:select/>
+    </f:entry>
+    </j:when>
+    <j:otherwise>
+      <f:invisibleEntry>
+        <f:select field="apiUri"/>
+      </f:invisibleEntry>
+    </j:otherwise>
+  </j:choose>
   <f:entry title="${%Credentials}" field="credentialsId">
     <c:select checkMethod="post"/>
   </f:entry>
-  <f:radioBlock name="isConfigurableByScan" value="scan" checked="${instance.isConfiguredByRepoScan()}" title="Github Repository Scan" inline="true">
+  <f:radioBlock name="isConfiguredByUrl" value="false" checked="${instance.configuredByUrl}" title="${%Repository Scan}" inline="true">
     <f:nested>
-      <j:choose>
-        <j:when test="${descriptor.apiUriSelectable}">
-          <f:entry title="${%API endpoint}" field="apiUri">
-            <f:select/>
-          </f:entry>
-        </j:when>
-        <j:otherwise>
-          <f:invisibleEntry>
-            <f:select field="apiUri"/>
-          </f:invisibleEntry>
-        </j:otherwise>
-      </j:choose>
-
       <f:entry title ="${%Owner}" field="repoOwner">
         <f:textbox/>
       </f:entry>
       <f:entry title="${%Repository}" field="repository">
         <f:select/>
       </f:entry>
-
     </f:nested>
   </f:radioBlock>
-  <f:radioBlock name="isConfigurableByScan" value="raw" checked="${!instance.isConfiguredByRepoScan()}" title="Github Repository HTTPs URL" inline="true">
+  <f:radioBlock name="isConfiguredByUrl" value="true" checked="${instance.configuredByUrl}" title="${%Repository URL}" inline="true">
     <f:nested>
-      <f:entry title="HTTPS URL" field="rawUrl">
+      <f:entry title="${%Repository URL}" field="repositoryUrl">
         <f:textbox/>
       </f:entry>
-
-<!--      <f:validateButton method="checkCredentials" title="Validate" with="rawUrl,credentialsId" />-->
+      <f:validateButton method="validateRepositoryUrlAndCredentials" title="${%Validate}" with="repositoryUrl,credentialsId" />
     </f:nested>
   </f:radioBlock>
 
-  <script language="javascript" type="text/javascript" src="${rootURL}/plugin/github-branch-source/show.js"/>
```

Comment:
I added this for showing the owner/repo info, otherwise is not visible

Reply:
It seemed to work ok when I was testing, but maybe that was because of what you noted in https://github.com/joseblas/github-branch-source-plugin/pull/1/files/fc55189cfb48583a64b1d6c4c337e63feb3f4e7e#r300705231 so I didn't test the problematic case?

It seems strange to me that the stuff in the radio block wouldn't be shown even with us setting `checked=${true}`, perhaps just a bug with heteroRadioList (the container for this whole file) in combination with nested radioBlocks?

Reply:
Maybe a dupe of a Jenkins core bug that I fixed in https://github.com/jenkinsci/jenkins/pull/3672? First released in Jenkins 2.145, though also in the 2.150 LTS. Right now this plugin only depends on 2.121.3, so it could be that issue.